microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -6,5 +6,143 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+{product-title} enables dynamic storage provisioning out of the box with the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider. The LVMS plugin is Red Hat's downstream version of TopoLVM--a CSI plugin for managing LVM volumes for Kubernetes. 
```

Comment:
```suggestion
{product-title} enables dynamic storage provisioning that is ready for immediate use with the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider. The LVMS plugin is Red Hat's downstream version of TopoLVM, a CSI plugin for managing LVM volumes for Kubernetes. 
```
See the [ISG guidance ](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#out-of-the-box) on _out of the box_ and [dashes](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-dashes). My understanding is that en dashes should only be used for ranges, and we do not use em dashes in technical docs.

Reply:
With the new rule on _plugin_ as the preferred term over _plug-in_, I recommend updating your H1 title on line 3 to _MicroShift storage plugin overview_.

Reply:
Also, I think that we aren't supposed to use possessives for product names, or to say something belongs to RH or IBM. WDYT? https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

Reply:
Got it